docker-compose-setup/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.." || exit
```

Comment:
Does this also work when executing the script from another folder than "docker-compose-setup"?

Reply:
Yes, this is the purpose of this line. In the first place I intended to make it runnable from the root folder.